### PR TITLE
Feat/security password

### PR DESCRIPTION
**Pull request by eliawk**

### CODE REVIEW

File: src/scripts/components/widgets/modals/ModalAskPassword.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+  //   })
+  //   if (password && password.length === 4) {
+  //     if (!this.state.isPassword) {
+  //       this.setState({
+  //         password: password,
+  //         isPassword: true
+  //       })
+  //     }
+  //   }
+  // }
```
Comment (bent0b0x) [inline src/scripts/components/widgets/modals/ModalAskPassword.js]: Delete this commented code?

File: src/scripts/components/widgets/modals/ModalCreatePassword.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+        sessionStorage.setItem(PASSWORD_TEMP, this.state.password)
+        this.props.openModal(MODAL.SHOW_SEED)
+      } else if (this.props.getContext === RESTORE_ACCOUNT) {
+        this.props.secureKeystore(this.state.password)
+        this.props.closeModal()
+      }
+    } else {
+      // Error, the two Passwords are different
+      this.setState({
+        error: `Hey, you have insert two different Passwords`
```
Comment (bent0b0x) [inline src/scripts/components/widgets/modals/ModalCreatePassword.js]: Maybe `Hey, your passwords do not match`

File: src/scripts/components/widgets/modals/ModalCreatePassword.js [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+  }
+
+  render () {
+    return (
+      <ModalContentWrapper>
+        <ModalScrollContent>
+          <Title>Create a password for securing your account</Title>
+          <Text small gray>
+            It will work for a password for fast transactions,{' '}
+            <strong>but only if you are logged in</strong>
```
Comment (bent0b0x) [inline src/scripts/components/widgets/modals/ModalCreatePassword.js]: What does this mean? I think we'll want to clean up the verbiage a bit here. Right now it is confusing

File: src/scripts/components/widgets/modals/ModalProfile.js [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+  render () {
+    return (
+      <ModalContentWrapper>
+        <ModalScrollContent>
+          <Title>Profile</Title>
+          <Text small gray>
+            This is how you be known on the Paratii network. Enter your email if
+            you like to receive updates and email about the community and your
+            actions.
+          </Text>
```
Comment (bent0b0x) [inline src/scripts/components/widgets/modals/ModalProfile.js]: How about:

```
This is how you'll be known on the Paratii network. Enter your email if you'd like to receive updates about the community and your activity.
```

File: src/scripts/components/widgets/modals/ModalShowSeed.js [hunk 5]
```diff
@@ -91,13 +93,13 @@ const WordIndex = styled.span`
...
-  margin-right: 10px;
-  width: 20px;
-`
+// const CopyButtonIcon = styled.svg`
+//   fill: ${Colors.purple};
+//   display: inline-block;
+//   height: 24px;
+//   margin-right: 10px;
+//   width: 20px;
+// `
```
Comment (bent0b0x) [inline src/scripts/components/widgets/modals/ModalShowSeed.js]: Delete these comments?

File: src/scripts/components/widgets/modals/ModalShowSeed.js [hunk 6]
```diff
@@ -179,27 +197,39 @@ class ModalShowSeed extends Component<Props, Object> {
...
+              I have copied the 12 words, they are secret, safe and sound
+            </RadioCheck>
+            {
+              // <Button
+              //   data-test-id="new-mnemonic-button"
+              //   onClick={this.copyWordsToClipboard}
+              //   gray
+              // >
+              //   <CopyButtonIcon>
+              //     <use xlinkHref="#icon-copy" />
```
Comment (bent0b0x) [inline src/scripts/components/widgets/modals/ModalShowSeed.js]: Commented

File: src/scripts/components/widgets/modals/ModalCreatePassword.js (hunk 3 above)
Comment (eliawk) [reply src/scripts/components/widgets/modals/ModalCreatePassword.js]: i've just copied the copy in the @pedrocasa UI!

File: src/scripts/actions/GlobalActions.js [hunk 7]
```diff
@@ -0,0 +1,7 @@
+/* @flow */
+
+import { createAction } from 'redux-actions'
+
+import { SET_CONTEXT } from 'constants/ActionConstants'
+
+export const setContext = createAction(SET_CONTEXT)
```
[the remark this answers is not in the document]
Comment (eliawk) [reply src/scripts/actions/GlobalActions.js]: i move it to `AppActions`

File: src/scripts/components/widgets/modals/ModalAskPassword.js (hunk 1 above)
Comment (eliawk) [reply src/scripts/components/widgets/modals/ModalAskPassword.js]: ok

**Pull request merged**